src/constants/CameraConstants.py

```diff
@@ -0,0 +1,29 @@
+import math
+
+from constants.RobotConstants import RobotConstants
+
+# Approximate measured Focal Lengths (FL is Focal length), at 640 x 480p
+MSLifeFL = 625
+LogitechFL = 800
+TargetSize = 13  # Inches
```

Comment:
Can move TargetSize to the dsvision file. It's not actually camera related anyways.